components/rating/rating.css

```diff
@@ -0,0 +1,3 @@
+.rating .fas{
+    color:var(--secondary-color);
```

Comment:
add cursor pointer property too

Reply:
ok, I will check that. Thank you for the review